optimum/exporters/neuron/utils.py

```diff
@@ -35,42 +67,88 @@
         from diffusers import ModelMixin, StableDiffusionPipeline
 
 
+class DiffusersPretrainedConfig(PretrainedConfig):
+    # override to update `model_type`
+    def to_dict(self):
+        """
+        Serializes this instance to a Python dictionary.
+
+        Returns:
+            :obj:`Dict[str, any]`: Dictionary of all the attributes that make up this configuration instance,
+        """
+        output = copy.deepcopy(self.__dict__)
+        return output
+
+
+def build_stable_diffusion_components_mandatory_shapes(
+    batch_size: Optional[int] = None,
+    sequence_length: Optional[int] = None,
+    num_channels: Optional[int] = None,
+    height: Optional[int] = None,
+    width: Optional[int] = None,
+    **kwargs,
```

Comment:
Why?

Reply:
Removed `**kwargs` as it's not necessary